bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/api/ApiHandler.java

```diff
@@ -360,5 +360,6 @@ public void closeSession() {
             logger.warn("Error closing session : {}", e.getMessage());
         }
         httpHeaders.remove(AUTH_HEADER);
+        managers.remove(LoginManager.class);
```

Comment:
This is just to be sure that a new session is tried with the new token when I change the bridge configuration (new token).
Maybe you should even fully clear managers ?

Reply:
Normalement, sur un dispose(), l'idée est de libérer les ressources, donc je pense qu'il faudrait plutôt complétement vider la map "managers".